.github/workflows/release-integration.yml

```diff
@@ -26,19 +26,20 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
         with:
-          submodules: 'true'
+          submodules: true
+      - name: Apply Sourcify patch customizations
```

Comment:
Q: By sourcify is this the server? When noted side by side it's strange one is Sourcify and the Other is Repository.

Reply:
I use a more consistent convention to name these steps and scripts.